lib/util/html.js

```diff
@@ -37,11 +34,14 @@ const frontendPage = ({ head='', body }) => html`
     <body>
       <div class="header"><a href="/">ODK Central</a></div>
       <div id="content">
-        ${body}
+        {{{body}}}
       </div>
     </body>
   </html>
 `;
+parse(template); // caches template for future perf.
+
+const frontendPage = ({ head='', body }) => render(template, { head, body });
```

Comment:
I [commented](https://github.com/getodk/central-backend/pull/981#discussion_r1335236657) in #981 that I think we should try to remove `frontendPage()` entirely. But that's probably a task for another PR.